azure-keyvault/upsertGroup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+groupName=$1
```

Comment:
I'm going to treat myself and add this workaround into the az mixin https://github.com/getporter/az-mixin/issues/32 so that eventually we don't need to tell people to use hacks like this. 😀 

It's fine for now, when I fix the mixin, I'll update the example.

Reply:
yeah its a work around I got from Ralph